debian/control

```diff
@@ -25,6 +24,9 @@ Homepage: https://github.com/intel/xpumanager
 Package: libxpum1
 Architecture: amd64
 Depends:
+  libze1 (>= 1.17.6),
+  intel-gsc (>=0.9.3),
```

Comment:
this might be libigsc and not intel-gsc.... need to check

Reply:
ldd /usr/bin/xpu-smi 
	linux-vdso.so.1 (0x00007d157db47000)
	libigsc.so.0 => /lib/x86_64-linux-gnu/libigsc.so.0 (0x00007d157d99f000)
	libxpum.so.1 => /lib/x86_64-linux-gnu/libxpum.so.1 (0x00007d157d400000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007d157d000000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007d157d8b2000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007d157d884000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007d157cc00000)
	libudev.so.1 => /lib/x86_64-linux-gnu/libudev.so.1 (0x00007d157d83d000)
	libze_loader.so.1 => /lib/x86_64-linux-gnu/libze_loader.so.1 (0x00007d157d33b000)
	libpciaccess.so.0 => /lib/x86_64-linux-gnu/libpciaccess.so.0 (0x00007d157d832000)
	/lib64/ld-linux-x86-64.so.2 (0x00007d157db49000)
	libcap.so.2 => /lib/x86_64-linux-gnu/libcap.so.2 (0x00007d157d825000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007d157d805000)

Looks like that's correct. I'll update it